### PR TITLE
Adjust assemble scripts as micropipenv is in UBI images

### PR DESCRIPTION
**Pull request by fridex**

### CODE REVIEW

File: ubi8-py36/s2i_assemble.patch [hunk 1]
```diff
@@ -1,47 +1,11 @@
...
-+echo "---> Installing dependencies via micropipenv ..."
-+micropipenv install --deploy
- 
- if [[ -f setup.py && -z "$DISABLE_SETUP_PY_PROCESSING" ]]; then
-   echo "---> Installing application ..."
+--- a/assemble	2020-08-26 08:36:24.689696930 +0200
++++ b/assemble	2020-08-26 08:38:32.765236073 +0200
+@@ -1,4 +1,7 @@
+ #!/bin/bash
++# Thoth's addition always uses micropipenv.
```
Comment (harshad16) [inline ubi8-py36/s2i_assemble.patch:5]: is the `+` required in these three lines.?

Comment (fridex) [reply ubi8-py36/s2i_assemble.patch:5]: Yes, it's a patch to assemble scripts.  The first `+` is from git patch, the second one is from the file itself.

Comment (fridex) [reply ubi8-py36/s2i_assemble.patch:5]: https://github.com/thoth-station/s2i-thoth/blob/951c03ffca9ce777118c6c42ad73a6a0ca491f20/ubi8-py36/Dockerfile#L28

Comment (fridex) [reply ubi8-py36/s2i_assemble.patch:5]: The build is successful locally for both ubi images.

Comment (harshad16) [reply ubi8-py36/s2i_assemble.patch:5]: thanks @fridex

**Review by harshad16 — APPROVED** [review]
Thanks for the changes :100:

**Pull request merged**